github/Repository.py

```diff
@@ -2304,6 +2312,18 @@ def get_teams(self):
             None
         )
 
+    def get_topics(self):
+        """
+        :calls: `GET /repos/:owner/:repo/topics <http://developer.github.com/v3/repos>`_
```

Comment:
Let's add the exact link:
https://developer.github.com/v3/repos/#list-all-topics-for-a-repository